api/src/main/java/org/pmiops/workbench/google/DirectoryServiceImpl.java

```
@@ -126,10 +120,11 @@ public User createUser(String givenName, String familyName, String username, Str
       .setEmails(new UserEmail().setType("custom").setAddress(contactEmail).setCustomType("contact"))
       .setChangePasswordAtNextLogin(true);
     retryHandler.run((context) -> getGoogleDirectoryService().users().insert(user).execute());
-    //This is a temporary solution until Mandrill is implemented.
-    //This way if one merge happens and the other is delayed for any reason,
-    //we won't break in a major way.
-    sendPasswordEmail(contactEmail, password, user);
+    try {
+      mailServiceProvider.get().sendEmail(contactEmail, password, user);
+    } catch (MessagingException e) {
+      return user;
```

Why are we swallowing the exception here?